src/map/handler/Map.TapHold.js

```diff
@@ -0,0 +1,108 @@
+import {Map} from '../Map';
+import {Handler} from '../../core/Handler';
+import * as DomEvent from '../../dom/DomEvent';
+import {Point} from '../../geometry/Point';
+import * as Util from '../../core/Util';
+import * as Browser from '../../core/Browser';
+
+/*
+ * L.Map.TapHold is used to simulate `contextmenu` event on long hold,
+ * which otherwise is not fired by mobile Safari.
+ */
+
+var tapHoldDelay = 600;
```

Comment:
I think we should add this as a option, there are feature request for this, so why not implementing?
Can be used from #7698

Reply:
There is a very little sense because we are unable to make such option work consistently across all the systems.
The best we can do - pick a value close to observable on real systems.

Reply:
Yes but we can still add it as undocumented option so if there is a reason why it should not `600` it is possible to change.

Reply:
Sure, but why expose it prematurely? We can do it exactly when we actualy need it.

Reply:
Because currently the releases happens not often, so when there would be a change needed the users can maybe wait over a year (I don't hope that this happens). If this is adedd now, the users can fix it self if Apple is changing something.

But I also understand it why it should not changeable. So it's ok for me.

Reply:
Out of curiosity, why was the timeout set to 1 second to start with?